package.json

```diff
@@ -15,7 +15,7 @@
     "test": "jest",
     "test:react": "pnpm test --prefix ./react test",
     "wsproxy": "node ./src/wsproxy/local_proxy.js",
-    "build": "rm -rf build/rollup && mkdir -p build/rollup/src/components && pnpm run copyindex && pnpm run copywc && pnpm run copywa && pnpm run copyresource && pnpm run copyconfig && rollup -c rollup.config.ts --configPlugin typescript && pnpm run --prefix ./react build:copy",
+    "build": "pnpm run --prefix ./backendai-ui build && rm -rf build/rollup && mkdir -p build/rollup/src/components && pnpm run copyindex && pnpm run copywc && pnpm run copywa && pnpm run copyresource && pnpm run copyconfig && rollup -c rollup.config.ts --configPlugin typescript && pnpm run --prefix ./react build:copy",
```

Comment:
I understand why we need it. in the near future, we should improve it support monorepo. But I guess. 'pnpm run --prefix ./backendai-ui build` would be placed before `pnpm run --prefix ./react` which make me more clear